github/src/main/kotlin/gropius/sync/github/Outgoing.kt

```diff
@@ -205,21 +283,108 @@ class Outgoing(
             }
         }
         val collectedMutations = mutableListOf<suspend () -> Unit>()
-        if (relevantTimeline.last() is ReopenedEvent) {
-            collectedMutations += githubReopenIssue(
-                imsProjectConfig,
+        if (shouldSyncType<ReopenedEvent, ClosedEvent>(
+                finalBlock, relevantTimeline, true
+            )
+        ) {
+            collectedMutations += githubReopenIssue(imsProjectConfig,
                 issueInfo,
                 finalBlock.map { it.lastModifiedBy().value })
         }
-        if (relevantTimeline.last() is ClosedEvent) {
-            collectedMutations += githubCloseIssue(
-                imsProjectConfig,
+        if (shouldSyncType<ClosedEvent, ReopenedEvent>(
+                finalBlock, relevantTimeline, false
+            )
+        ) {
+            collectedMutations += githubCloseIssue(imsProjectConfig,
```

Comment:
you sure the autoformatter did this?

Reply:
oscillates between this and 
```kt
            collectedMutations += githubCloseIssue(
                imsProjectConfig,
                issueInfo,
                finalBlock.map { it.lastModifiedBy().value })
```
every few formats

Reply:
strange